ui/raidboss/emulator/data/AnalyzedEncounter.ts

```diff
@@ -213,14 +213,11 @@ export default class AnalyzedEncounter extends EventBus {
       if (!perspective || !triggerHelper)
         throw new UnreachableCode();
 
-      const delay = currentTriggerStatus.delay ?? 0;
-
       perspective.triggers.push({
         triggerHelper: triggerHelper,
         status: currentTriggerStatus,
         logLine: log,
-        resolvedOffset: (log.timestamp - this.encounter.startTimestamp) +
-          (delay * 1000),
+        resolvedOffset: (log.timestamp - this.encounter.startTimestamp),
```

Comment:
Sorry for the confusion on my part, but I don't think I understand this change.  I could understand adjusting for the encounter's offset, but I don't understand how you can take out the individual trigger's delay.  Could you explain the change to me?

Reply:
Oh sorry, I can't read.  I see your comment, re: `Also this PR fixes a bug I noticed where triggers with a delaySeconds parameter were getting additional time added to their resolvedOffset due to leftover code from a previous way of calculating the resolvedOffset value.`